src/tex.js

```diff
@@ -203,8 +231,24 @@ MathJax.Hub.Register.StartupHook('TeX Jax Ready', function () {
       return token;
     },
     arabicNumber: (function () {
-      var englishNumbersRegExp = /[0-9]/g;
-      var numbersMap = MathJax.Hub.config.Arabic.numbersMap;
+      // check for MathJax options
+      useHindiNumeral = MathJax.Hub.config.useHindiNumeral
+      useArabicDecimalSeparator = MathJax.Hub.config.useArabicDecimalSeparator
+      
+      // Default options
+      if (useHindiNumeral == null) useHindiNumeral = true
+      if (useArabicDecimalSeparator == null) useArabicDecimalSeparator = true
+
+      arConfig = MathJax.Hub.config.Arabic
+      if(useArabicDecimalSeparator){
+        var englishNumbersRegExp = /[0-9]\.?/g;
+        var numbersMap = {...arConfig.numbersMap, ...arConfig.arabicDecimalSeparator}
```

Comment:
I like modernizing the code but. Not sure if we have the build steps to support such new syntax.

I'm inclined to revert to ES5 syntax (i.e. just regular JS), and save any efforts for upgrading to MathJax v3. See https://github.com/mathjax/mathjax-v3/pull/136 for more info.

Reply:
Changed the spread syntax to es5